uber/site_sections/preregistration.py

```diff
@@ -301,6 +301,10 @@ def process_free_prereg(self, session):
 
     @credit_card
     def prereg_payment(self, session, payment_id, stripeToken):
```

Comment:
The error we were seeing was because this function was called WITHOUT the parameters at all. So you need to add defaults here:
```
def prereg_payment(self, session, payment_id=None, stripeToken=None):
```